sketch_map_tool/templates/digitize.html

```diff
@@ -48,6 +48,7 @@
             </div>
             <div>
                 <p>
+                    <input type="checkbox" id="consent" name="consent" value="True" checked>
```

Comment:
the id/name shouldn't just be `consent`. Ideally it would indicate what you consent to.